src/pages/index.tsx

```diff
@@ -2,24 +2,32 @@
 /* eslint-disable @next/next/no-img-element */
 import { NextPage } from "next";
 import Link from "next/link";
+import Image from "next/image";
```

Comment:
This is wrong!!. You should do `../../public...` instead use `/[path]` eg `/images`

Reply:
Sir, doing this gave me an error that the module was not found